src/main/java/org/sciborgs1155/lib/FaultLogger.java

```diff
@@ -280,8 +282,17 @@ public static boolean check(CANSparkBase spark) {
    * @return If the spark is working without errors.
    */
   public static boolean check(CANSparkBase spark, REVLibError error) {
+    Map<Integer, String> nickname = new HashMap<>();
```

Comment:
I didn't actually think of this! This is a good idea, though it would be hard to maintain as there would be many maps in different places based on different pieces of hardware that might also need nicknames (like the register method for talons that should exist now or in the future.) 

It could be moved out of the method into the class, but I think nicknames will likely be used outside of `FaultLogger`, like for giving motors specific names through logging on NT for more convenient debugging.

Instead, change of plans: I personally think it would work and look best to create a static map in `Ports.java` that `check()` and `register()` access to retrieve nicknames according to corresponding IDs. They could be easily referenced and would exist in one place, making updates easy. 

LMK if you have concerns